.github/workflows/test.yml

```diff
@@ -42,6 +42,32 @@ jobs:
             token: ${{ secrets.CODECOV_TOKEN }}
             slug: pasqal-io/qubo-solver
 
+  unit_tests_windows:
+    name: Unit testing Windows
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        python-version: ["3.9", "3.10", "3.11", "3.12"]
```

Comment:
I think that latest qoolqit doesn't support 3.9 anymore, no?

Just taking note that we'll need to specify if/when we drop compatibility with 3.9 in the release notes.